core/src/main/scala/chisel3/properties/Property.scala

```diff
@@ -323,6 +323,9 @@ sealed trait Property[T] extends Element { self =>
 
   final def +(that: Property[T])(implicit ev: PropertyArithmeticOps[Property[T]], sourceInfo: SourceInfo): Property[T] =
     ev.add(this, that)
+
+  final def *(that: Property[T])(implicit ev: PropertyArithmeticOps[Property[T]], sourceInfo: SourceInfo): Property[T] =
+    ev.mul(this, that)
```

Comment:
can you copy the great info from `properties.md` as scaladoc here?

Reply:
Ah, yes I'll add scaladoc to the public methods, thanks for pointing it out.